test/kokoro_linux_aarch64.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright 2021 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+cd $(dirname $0)/..
+
+test/aarch64/prepare_qemu.sh
+
+# better update submodules here. We could update the submodule when running
+# under an emulator as well, but it comes with performance penalty.
+git submodule update --init --recursive
+
+if [[ -t 0 ]]; then
+  DOCKER_TTY_ARGS="-it"
+else
+  # The input device on kokoro is not a TTY, so -it does not work.
+  DOCKER_TTY_ARGS=
+fi
+
+# the test command to run under an emulated aarch64 docker container.
+# we only run tests for a single version of node, since tests under an emulator are significantly slower.
+TEST_NODE_COMMAND="node_versions='8' ./run-tests.sh"
```

Comment:
Node 8 is past EOL. We still support it, but if we can just choose one version to test on, this one shouldn't be the priority. I think it would be best to future proof it and use the latest version, `16`.